python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -722,6 +722,19 @@ def test_write_operations(self):
         ndf = self.connect.read.table("parquet_test")
         self.assertEqual(set(df.collect()), set(ndf.collect()))
 
+    def test_agg_with_avg(self):
+        """SPARK-41325: groupby.avg()"""
```

Comment:
```suggestion
        # SPARK-41325: groupby.avg()
```

per [contribution guide ](https://spark.apache.org/contributing.html).

Reply:
Done